app/Http/Controllers/Three/SignupsController.php

```diff
@@ -102,6 +102,12 @@ public function index(Request $request)
      */
     public function show(Request $request, Signup $signup)
     {
+        if ($request->query('include') === 'posts') {
```

Comment:
@DFurnes also for this one, since the query happens later when we send to the transformer, and just includes all posts if it sees the `includes=posts` flag, not sure how to weed out the ones with the status not approved?

Reply:
figured it out - disregard!